code/modules/item_worth/worths_list.dm

```diff
@@ -25,21 +25,17 @@ var/list/worths = list(
 					/obj/item/weapon/reagent_containers/glass = -60,
 					/obj/item/weapon/reagent_containers = -1,
 //GUNS,
-					/obj/item/weapon/gun/energy/laser/practice = 500,
-					/obj/item/weapon/gun/energy/laser/dogan = 1900,
-					/obj/item/weapon/gun/energy/laser = 3000,
-					/obj/item/weapon/gun/energy/retro = 1400,
-					/obj/item/weapon/gun/energy/captain = 2500,
-					/obj/item/weapon/gun/energy/lasercannon = 4100,
-					/obj/item/weapon/gun/energy/xray = 2900,
-					/obj/item/weapon/gun/energy/sniperrifle = 5400,
+					/obj/item/weapon/gun/projectile/linecutter = 70,
+					/obj/item/weapon/gun/projectile/seeker = 60,
+					/obj/item/weapon/gun/projectile/ripper = 60,
+					/obj/item/weapon/gun/energy/forcegun = 60,
+					/obj/item/weapon/gun/energy/contact = 60,
+					/obj/item/weapon/gun/projectile/divet = 40,
+					/obj/item/weapon/gun/energy/cutter = 40,
```

Comment:
I'm fairly certain that this does not do what you anticipate it does, as it the regular energy gun was defined as 3000 and I can't see these being used anywhere in the cargo console code, rather it seems these were used to define prices for traders (I think vox) on bay judging by the blame history. Has this been tested? Cargo on bay has always been very rudimentary and hasn't taken things other than phoron (and apparently mining scan disks) for ages.

Reply:
This is the exports list. I'm fairly sure this is the worths list for exports. You're able to export things out. Haven't tested yet.